Assets/Scripts/Controllers/Events/TradeController.cs

```diff
@@ -54,18 +54,12 @@ public void CallTradeShipTest(Furniture landingPad)
         go.transform.parent = WorldController.Instance.transform;
         TraderShipController controller = go.AddComponent<TraderShipController>();
         TradeShips.Add(controller);
-        controller.Trader = trader;
-        controller.Speed = 5f;
         go.transform.position = new Vector3(-10, 50, 0);
-        controller.LandingCoordinates = new Vector3(landingPad.Tile.X + 1, landingPad.Tile.Y + 1, 0);
-        controller.LeavingCoordinates = new Vector3(100, 50, 0);
         go.transform.localScale = new Vector3(1, 1, 1);
         SpriteRenderer spriteRenderer = go.AddComponent<SpriteRenderer>();
         spriteRenderer.sprite = SpriteManager.GetSprite("Trader", prototype.AnimationIdle.CurrentFrameName);
         spriteRenderer.sortingLayerName = "TradeShip";
-        controller.AnimationFlying = prototype.AnimationFlying.Clone();
-        controller.AnimationIdle = prototype.AnimationIdle.Clone();
-        controller.Renderer = spriteRenderer;
+        controller.Init(leavingCoords: new Vector3(100, 50, 0), landingCoords: new Vector3(landingPad.Tile.X + 1, landingPad.Tile.Y + 1, 0), speed: 5f, trader: trader, animationIdle: prototype.AnimationIdle.Clone(), animationFlying: prototype.AnimationFlying.Clone(), renderer: spriteRenderer);
```

Comment:
Put a `TODO:` here, it looks like too much work is being put on the caller of the tradeship, and I suspect more of the work could be done in init or elsewhere in the controller, so that calling a tradeship is simpler on the calling side. Something to the effect of `// TODO: Look into passing more of the work of calling a tradeship off to the controller`. I think it's outside the scope of this PR, but I'd prefer one of to not have to waste time digging back through the code to find it again, or just forget about it completely.